src/main/java/seedu/address/logic/parser/TagEventCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.TagEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new TagEventCommand object
+ */
+public class TagEventCommandParser implements Parser<TagEventCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagPersonsToEventCommand
+     * and returns an TagPersonsToEventCommand object for execution.
```

Comment:
Should be `TagEventCommand`

```suggestion
     * Parses the given {@code String} of arguments in the context of the TagEventCommand
     * and returns an TagEventCommand object for execution.
```